prereq_tools/base.py

```diff
@@ -64,6 +64,132 @@
 else:
     import configparser as ConfigParser
 
+class EnvironmentModule():
+    """Class for utilizing Modules component to load environment modules"""
+
+    def __init__(self):
+        """Load Modules for initializing envirables"""
+        self._module_func = lambda *x: False
+        func = self._init_module_func()
+        if func:
+            self._module_func = func
```

Comment:
(pylint-redefined-variable-type) Redefinition of self._module_func type from function to instancemethod